homework/CSharpOOP/Program.cs

```diff
@@ -0,0 +1,127 @@
+﻿#region Task_1
+//Console.WriteLine("Введите количество предметов, цену за единицу и скидку");
+//var numberOfItems = double.Parse(Console.ReadLine());
+//var price = double.Parse(Console.ReadLine());
+//var discount = double.Parse(Console.ReadLine());
+//var totalPrice = new Homework_1().GetDiscount(numberOfItems, price, discount);
+//Console.WriteLine($"Итоговая стоимость: {totalPrice}");
+#endregion
+#region Task_2
+using System.Diagnostics;
+
+//Console.WriteLine("Enter the product, its quantity and unit price");
+//string Item = Console.ReadLine();
+//var Number = double.Parse(Console.ReadLine());
+//var Price = double.Parse(Console.ReadLine());
+//Console.WriteLine("Enter the product, its quantity and unit price");
+//string secondItem = Console.ReadLine();
+//var secondNumber = double.Parse(Console.ReadLine());
+//var secondPrice = double.Parse(Console.ReadLine());
+//Console.WriteLine("Enter the product, its quantity and unit price");
+//string thirdItem = Console.ReadLine();
+//var thirdNumber = double.Parse(Console.ReadLine());
+//var thirdPrice = double.Parse(Console.ReadLine());
+//Console.WriteLine("Enter discount");
+//var discount = double.Parse(Console.ReadLine());
+//Console.WriteLine(new Homework_1().GetDiscountForNamedItem(Item, Price, discount));
+//Console.WriteLine(new Homework_1().GetDiscountForNamedItem(secondItem, secondPrice, discount););
+//Console.WriteLine(new Homework_1().GetDiscountForNamedItem(thirdItem, thirdPrice, discount););
+//Console.WriteLine($"Total payable: {new Homework_1().GetDiscountItem(Number, Price, secondNumber, secondPrice, thirdNumber, thirdPrice, discount)}");
```

Comment:
Why do you need to overload method if you can just get 3 results using 1 method that you have already created, then just summariza them and get final count. Your previous code was okay, you just needed to use created method in order to get 3 total sums :)